Makefile

```diff
@@ -11,7 +11,7 @@ default: bin binaries
 all: bin binaries universo
 
 .PHONY: binaries
-binaries: dkcheck.native dktop.native dkdep.native dkprune.native dkmeta.native dkpretty.native
+binaries: dkcheck.native dktop.native dkprune.native dkmeta.native
```

Comment:
Do you plan to remove dkdep?? I use it for testing lp2dk.

Reply:
I guess, since it is replaced with `dkcheck dep`. An update will be required on lp2dk's side.